src/PIL/Image.py

```diff
@@ -744,6 +744,16 @@ def __array_interface__(self) -> dict[str, str | bytes | int | tuple[int, ...]]:
         new["shape"], new["typestr"] = _conv_type_shape(self)
         return new
 
+    def __arrow_c_schema__(self) -> object:
+        self.load()
+        return self.im.__arrow_c_schema__()
+
+    def __arrow_c_array__(
+        self, requested_schema: object | None = None
+    ) -> Tuple[object, object]:
```

Comment:
It's directly from here: https://arrow.apache.org/docs/format/CDataInterface/PyCapsuleInterface.html#protocol-typehints

Reply:
pyarrow still supports Python 3.8 - https://pypi.org/project/pyarrow/
Since we don't, it can be updated - https://peps.python.org/pep-0585/#implementation